openslides_backend/action/relations/relation_manager.py

```diff
@@ -93,14 +125,16 @@ def get_relation_updates(
                         "action": action,
                     }
                 )
+        return calculated_field_handler_calls
 
-        self.apply_relation_updates(relations)
-
+    def handle_calculated_fields_2(
```

Comment:
please rename, something like `execute_calculated_field_handler_calls` might be more appropriate

Reply:
Renamed.